metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/EnrichmentWorker.java

```diff
@@ -27,8 +27,9 @@
  */
 public class EnrichmentWorker {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(EnrichmentWorker.class);
-
+  private static final Logger LOGGER = LogManager.getLogger(EnrichmentWorker.class);
```

Comment:
We should not replace slf4j with log4j. The old version of the statement was the right one.

Reply:
This is required for the reason mentioned previously: to change log levels during runtime.